Yosemite/Yosemite/Model/Storage/OrderItem+ReadOnlyConvertible.swift

```diff
@@ -26,16 +26,19 @@ extension Storage.OrderItem: ReadOnlyConvertible {
     /// Returns a ReadOnly version of the receiver.
     ///
     public func toReadOnly() -> Yosemite.OrderItem {
+        let orderItemTaxes = taxes?.map { $0.toReadOnly() } ?? [Yosemite.OrderItemTax]()
+
         return OrderItem(itemID: Int(itemID),
                          name: name ?? "",
                          productID: Int(productID),
                          variationID: Int(variationID),
-                         quantity: quantity,
+                         quantity: quantity as Decimal,
                          price: price ?? NSDecimalNumber(integerLiteral: 0),
                          sku: sku,
                          subtotal: subtotal ?? "",
                          subtotalTax: subtotalTax ?? "",
                          taxClass: taxClass ?? "",
+                         taxes: orderItemTaxes,
```

Comment:
would the diffs in this class make sense to be in the Storage PR? not sure if these might fix the CI in https://github.com/woocommerce/woocommerce-ios/pull/1363

Reply:
It would fix the Storage PR, yes. That's why I don't merge the Networking and Storage layer PRs until the Yosemite one passes all tests. I merge all 3, one after another, so that way `develop` does not break after all the merges are completed. It muddies the code reviews to add Yosemite layer changes to the Networking and Storage layers. (As you saw when I mistakenly added Networking changes to this PR).

Is there a different approach that would make you more comfortable?